README.md

```diff
@@ -32,7 +32,7 @@ steps:
     uses: ridedott/release-me-action@master
     with:
       # Configure semantic release to run from a specific branch.
-      # Default: master
+      # Default: @semantic-release/semantic-release Javascript API branches parameter default value
```

Comment:
Can we add some link to their documentation, so it's easier to find?

Reply:
Done!